webhook/resources/volume/mutator.go

```diff
@@ -189,8 +191,13 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 			return nil, werror.NewInvalidError(fmt.Sprintf("get invalid size for volume %v: %v", backup.Status.VolumeSize, err), "")
 		}
 
-		moreLabels[types.LonghornLabelBackupVolume] = bvName
+		moreLabels[types.LonghornLabelBackupVolume] = canonicalBVName
+	}
+	if backupTargetName == "" {
+		backupTargetName = types.DefaultBackupTargetName
```

Comment:
For a newly created volume, the v.spec.backupTargetName is always empty.
Can we add parameter `backupTargetName` in the storageClass, then the field is set from the parameter?